### DISCUSSION TITLE
Could not open LXMF link

### DISCUSSION THREAD

**Discussion by cascafico** [markqvist/Reticulum#608] [Nomad Network]
Hello Everyone.

It happens whenever I click on lxmf addresses in .mu pages; i.e. clicking on "Contact" at page 42464efb1d4fe2615c1016e24c3a7c86:/page/index.mu 

nomadnet reports in footer:
`Could not open LXMF link: 'utf-8' codec can't decode byte 0x92 in position 0: invalid start byte`

**Comment by markqvist** [comment]
Thanks! That was a bug. Fixed in https://github.com/markqvist/NomadNet/commit/dc43bc6a22b5be668a3d75be89c5bae0725b1af5.
